engine/flink/components/base/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/aggregates.scala

```diff
@@ -140,24 +139,23 @@ object aggregates {
   }
 
   object LastAggregator extends Aggregator {
-
-    override type Aggregate = AnyRef
+    override type Aggregate = Option[AnyRef]
 
     override type Element = AnyRef
 
-    override def zero: Aggregate = null
+    override def zero: Aggregate = None
 
     override def isNeutralForAccumulator(element: LastAggregator.Element, currentAggregate: LastAggregator.Aggregate): Boolean = false
 
-    override def addElement(el: Element, agg: Aggregate): Aggregate = el
+    override def addElement(el: Element, agg: Aggregate): Aggregate = Some(el)
 
-    override def mergeAggregates(agg1: Aggregate, agg2: Aggregate): Aggregate = agg2
+    override def mergeAggregates(agg1: Aggregate, agg2: Aggregate): Aggregate = agg2.orElse(agg1)
```

Comment:
@mproch we won't get here NPE for old states with null? Or `addElement` is invoked immediately after `zero` in this case?